org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/ClassFileComparatorTest.java

```diff
@@ -368,27 +368,6 @@ public void test011() {
 			removeTempClass("A011");
 		}
 	}
-	public void test012() {
-		try {
-			String sourceA012 =
-				"public class A012 {\n" +
-				"public Class foo() {\n" +
-				"\treturn null;\n" +
-				"}\n" +
-				"}";
-			compileAndDeploy(sourceA012, "A012");
-			String sourceA012_2 =
-				"public class A012_2 {\n" +
-				"public Class foo() {\n" +
-				"\treturn A012_2.class;\n" +
-				"}\n" +
-				"}";
-			compileAndDeploy(sourceA012_2, "A012_2");
-			assertTrue(areStructurallyDifferent("A012", "A012_2", false, false));
-		} finally {
-			removeTempClass("A012");
-		}
-	}
```

Comment:
Not immediately clear why this test is deleted ? It looks very similar to the test that follows - is that the reason ? Still we may have retained it as some parameters are different ?

Reply:
Removal was requested by Stephan [in this comment](https://github.com/eclipse-jdt/eclipse.jdt.core/pull/2551#discussion_r1685511582).
Quote:

> previously, we tested that at 1.4- we translated the class literal into a synthetic field. Since 1.5 this is > no longer needed (class literals now participate in ldc).
> I vote for removing this test.